composer.json

```
@@ -10,12 +10,11 @@
         }
     ],
     "require": {
-        "php": "^7.1 || ^8.0",
+        "php": "^7.2 || ^8.0",
         "ext-json": "*",
         "imagine/imagine": "^0.7.1 || ^1.0",
-        "symfony/filesystem": "^2.8 || ^3.0 || ^4.0 || ^5.0",
-        "symfony/polyfill-php73": "^1.11",
-        "webmozart/path-util": "^2.0"
+        "symfony/filesystem": "^5.4",
```

Can we use Symfony 6 here already?

```suggestion
        "symfony/filesystem": "^5.4 || ^6.0",
```